docs/faq/help.txt

```diff
@@ -22,11 +22,13 @@ Then, please post it in one of the following channels:
 * The Django Forum section `"Using Django"`_. This is for web-based
   discussions.
 * The |django-users| mailing list. This is for email-based discussions.
+* The `Django Discord server`_ for chat-based discussions.
```

Comment:
I noticed that the Django Forum recommended the "Using Django" section. Do we want to recommend a channel for getting help? I know the Discord does "welcome" you a bit so might not be needed 🤔 
What do you think? (cc @knyghty)

Reply:
There is a #getting-started channel but it's probably not the right channel for _everyone_ coming through here. The channels are also a bit more subject to change than the forum, so probably best to not point to anything specific but let the server onboarding flow do the work.